common/djangoapps/util/date_utils.py

```diff
@@ -52,7 +53,7 @@ def get_time_display(dtime, format_string=None, coerce_tz=None):
         try:
             to_tz = timezone(coerce_tz)
         except UnknownTimeZoneError:
-            to_tz = UTC
+            to_tz = utc
         dtime = to_tz.normalize(dtime.astimezone(to_tz))
```

Comment:
Correction: I checked the pytz docs; normalizing won't cause any problems.  It's just unnecessary in this case, because no timedelta math has been performed on the datetime object after converting it to `to_tz`.

This code is slightly inefficient but harmless.  Feel free to leave it alone.